corehq/apps/app_manager/models.py

```diff
@@ -1500,8 +1500,13 @@ def _assert_unexpected_default_media_call(self, media_attr):
         assert media_attr in ('media_image', 'media_audio')
         media = getattr(self, media_attr)
         if isinstance(media, dict) and list(media) == ['default']:
+            from corehq.util.view_utils import get_request
+            request = get_request()
+            url = ''
+            if request:
+                url = request.META.get('HTTP_REFERER')
```

Comment:
TIL this misspelling is [in the HTTP standard](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.36) :man_facepalming:

Reply:
aww, that's a shame

Reply:
Unrelated, a lot of emoticons just show up as boxes in my email, but that particular one shows as a box and a male symbol, which always confuses me for half a second:

<img width="561" alt="screen shot 2018-03-30 at 10 59 23 am" src="https://user-images.githubusercontent.com/1486591/38142249-9aa24c90-3409-11e8-8aa0-f717d595d3a7.png">

Reply:
I would react to this with a facepalm emoji but I fear that would only exacerbate the situation